cmd/mimir/help.txt.tmpl

```diff
@@ -891,6 +895,380 @@ Usage of ./cmd/mimir/mimir:
+    	Enable AWS Server Side Encryption. Supported values: SSE-KMS, SSE-S3.
+  -usage-tracker.snapshots-storage.s3.sts-endpoint string
+    	Accessing S3 resources using temporary, secure credentials provided by AWS Security Token Service.
+  -usage-tracker.snapshots-storage.storage-prefix string
+    	Prefix for all objects stored in the backend storage. For simplicity, it may only contain digits and English alphabet letters.
+  -usage-tracker.snapshots-storage.swift.application-credential-id string
+    	OpenStack Swift application credential id
+  -usage-tracker.snapshots-storage.swift.application-credential-name string
+    	OpenStack Swift application credential name
+  -usage-tracker.snapshots-storage.swift.application-credential-secret string
+    	OpenStack Swift application credential secret
+  -usage-tracker.snapshots-storage.swift.auth-url string
+    	OpenStack Swift authentication URL
+  -usage-tracker.snapshots-storage.swift.auth-version int
+    	OpenStack Swift authentication API version. 0 to autodetect.
+  -usage-tracker.snapshots-storage.swift.container-name string
+    	Name of the OpenStack Swift container to put chunks in.
+  -usage-tracker.snapshots-storage.swift.domain-id string
+    	OpenStack Swift user's domain ID.
+  -usage-tracker.snapshots-storage.swift.domain-name string
+    	OpenStack Swift user's domain name.
+  -usage-tracker.snapshots-storage.swift.password string
+    	OpenStack Swift API key.
+  -usage-tracker.snapshots-storage.swift.project-domain-id string
+    	ID of the OpenStack Swift project's domain (v3 auth only), only needed if it differs the from user domain.
+  -usage-tracker.snapshots-storage.swift.project-domain-name string
+    	Name of the OpenStack Swift project's domain (v3 auth only), only needed if it differs from the user domain.
+  -usage-tracker.snapshots-storage.swift.project-id string
+    	OpenStack Swift project ID (v2,v3 auth only).
+  -usage-tracker.snapshots-storage.swift.project-name string
+    	OpenStack Swift project name (v2,v3 auth only).
+  -usage-tracker.snapshots-storage.swift.region-name string
+    	OpenStack Swift Region to use (v2,v3 auth only).
+  -usage-tracker.snapshots-storage.swift.user-domain-id string
+    	OpenStack Swift user's domain ID.
+  -usage-tracker.snapshots-storage.swift.user-domain-name string
+    	OpenStack Swift user's domain name.
+  -usage-tracker.snapshots-storage.swift.user-id string
+    	OpenStack Swift user ID.
+  -usage-tracker.snapshots-storage.swift.username string
+    	OpenStack Swift username.
+  -usage-tracker.target-snapshot-file-size-bytes int
+    	Target size of a snapshot file in bytes. This is used to determine when to create a new snapshot file. It should be greater than 0. (default 104857600)
+  -usage-tracker.use-global-series-limits
+    	If true, the usage-tracker service will use global in-memory series limits instead of the active series limits. This is useful for testing purposes only.
```

Comment:
```suggestion
    	If true, the usage-tracker service uses global in-memory series limits instead of the active series limits. This is useful for testing purposes only.
```